appcelerator/titanium_mobile
# [TIMOB-26227] Android: Removed kroll thread support

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-26227

**Summary:**
- The "tiapp.xml" property "run-on-main-thread" is now ignored.
- Titanium's JavaScript runtime can now only run on the main UI thread.

**Test:**
1. Download "kitchensink-v2" project from [here](https://github.com/appcelerator/kitchensink-v2).
2. Build and run on Android.
3. Verify that all UI features in kitchen sink work without issue. _(Note that Google Maps, Facebook, and Twitter features can be skipped since they require an app ID.)_
